horace_core/sqw/@sqw/coordinates_calc.m

```diff
@@ -0,0 +1,189 @@
+function w = coordinates_calc(w, name)
+% Replace the sqw's signal and variance data with coordinate values (see below)
```

Comment:
should be worth commenting that this is the function formerly known as signal, for traceability.

Also at this point "the sqw" is not defined; you mean w. Can we also be a DND? (see comment below)

Reply:
This is a method on `sqw`. "the sqw" is the `sqw` you called this from.

Will add reference to signal.